### PR TITLE
Update lumi.json

### PR DESCRIPTION
**Pull request by ghost**
Website: https://lumidocs.vercel.app/

### CODE REVIEW

File: domains/_vercel.lumi.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
     "discord": "563697359423406082"
   },
   "record": {
-    "TXT": "vc-domain-verify=lumi.is-a.dev,38de86570893517719b8"
+    "A": [76.76.21.21]
```
Comment (vintheweirdass) [inline domains/_vercel.lumi.json:7]: this is incorrect. If you want to add the vercel IP, do that on `lumi.json` instead

Comment (vintheweirdass) [inline domains/_vercel.lumi.json:7]: this is incorrect. If you want to add the vercel IP, do that on `lumi.json` instead

Comment (itsFatlum) [reply domains/_vercel.lumi.json]: @vintheweirdass done

File: domains/lumi.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
     "discord": "563697359423406082"
   },
   "record": {
+    "A": [76.76.21.21],
```
Comment (Stef-00012) [inline domains/lumi.json:8]: This should be a string
```suggestion
    "A": ["76.76.21.21"],
```

**Review by Stef-DP — DISMISSED** [review]

**Comment by itsFatlum** [comment]
Now everything should be right 😭

**Review by iostpa — APPROVED** [review]
Seems fine to me now

**Pull request merged**